tools/ci_build/get_docker_image.py

```diff
@@ -115,16 +117,41 @@ def main():
             args.dockerfile,
             args.context,
         )
+    elif args.use_imagecache:
+        log.info("Building image with pipeline cache...")
+        run(
+            args.docker_path,
+            "--log-level",
+            "error",
+            "buildx",
+            "build",
+            "--tag",
+            full_image_name,
+            "--cache-from",
+            full_image_name,
+            "--build-arg",
+            "BUILDKIT_INLINE_CACHE=1",
```

Comment:
"BUILDKIT_INLINE_CACHE=1" was added for remote caching. Do you still need it?

Reply:
Thanks. updated.

Reply:
Yes, it's needed. 
Without it, even the cache is stored, the building will be triggered.
https://stackoverflow.com/questions/73161556/how-do-i-know-if-docker-image-build-with-inline-cache